plato/api.py

```diff
@@ -259,8 +261,7 @@ def update_template(template_id: str):
             db.session.commit()
 
             # uploads template files from zip file to S3
```

Comment:
Can you update these comments? They specifically mention S3

Reply:
ah oops, I missed this! good catch